pkgs/development/python-modules/python-gnupg/default.nix

```
@@ -1,16 +1,25 @@
-{ stdenv, buildPythonPackage, fetchPypi, gnupg }:
+{ stdenv, buildPythonPackage, fetchpatch, fetchPypi, gnupg }:
 
 buildPythonPackage rec {
   pname   = "python-gnupg";
-  version = "0.4.6";
+  version = "0.4.6"; # Please remove the patch below when >0.4.6
 
   src = fetchPypi {
     inherit pname version;
     sha256 = "3aa0884b3bd414652c2385b9df39e7b87272c2eca1b8fcc3089bc9e58652019a";
   };
 
+  patches = [
+  ] ++ stdenv.lib.optionals (let cutoff = "0.4.6"; in version == cutoff || stdenv.lib.versionOlder version cutoff) [
```

optionals will return `[ ]` if the condition is false
```suggestion
  patches = stdenv.lib.optionals (let cutoff = "0.4.6"; in version == cutoff || stdenv.lib.versionOlder version cutoff) [
```

---

Yes but this way is more diff friendly. But I don't really care much.